<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1661</width>
    <height>937</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>1400</x>
      <y>20</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="map">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1671</width>
      <height>921</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/img/img/background.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="road1hor">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>631</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="road0hor">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>631</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="road2hor">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>631</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="road3hor">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>631</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="road5hor">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>510</y>
      <width>631</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="road4hor">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>631</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="road0ver">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>140</y>
      <width>20</width>
      <height>131</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <zorder>map</zorder>
   <zorder>horizontalSlider</zorder>
   <zorder>horizontalSlider</zorder>
   <zorder>road1hor</zorder>
   <zorder>road0hor</zorder>
   <zorder>road2hor</zorder>
   <zorder>road3hor</zorder>
   <zorder>road5hor</zorder>
   <zorder>road4hor</zorder>
   <zorder>road0ver</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="meun">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
